apps/events/views.py

```diff
@@ -1,18 +1,27 @@
+import datetime
+
 from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
 from django.core.exceptions import PermissionDenied
 from django.shortcuts import get_object_or_404, render
 from django.urls import reverse_lazy
 from django.views.generic import CreateView, DeleteView, UpdateView
+from django_ical.views import ICalFeed
 
 from apps.events.forms import EventForm
 from apps.events.models import Event
 
 
 def events_list(request):
     if request.user.is_authenticated:
-        events_list = Event.objects.order_by("start_date_time").all()
+        events_list = (
+            Event.objects.order_by("start_date_time").filter(end_date_time__gte=datetime.datetime.today()).all()
+        )
     else:
-        events_list = Event.objects.filter(member_only=False).order_by("start_date_time")
+        events_list = (
+            Event.objects.filter(member_only=False)
+            .order_by("start_date_time")
+            .filter(end_date_time__gte=datetime.datetime.today())
+        )
```

Comment:
Do we not want previous events to be able to be seen at all? Maybe sth similar to the photos section that allows them to still be seen when going to the past? Cause only showing upcoming events at first makes sense, but i feel there is a better solution then removing them altogether.